chromium_src/chrome/install_static/brave_install_util_unittest.cc

```diff
@@ -9,11 +9,11 @@
 #include "base/macros.h"
 #include "base/stl_util.h"
 #include "base/test/test_reg_util_win.h"
+#include "chrome/chrome_elf/nt_registry/nt_registry.h"
```

Comment:
don't we also need to add a dep for `//chrome/chrome_elf` to `test/BUILD.gn` for these test files?

Reply:
`test/BUILD.gn`'s target `brave_install_static_unittests` that includes this unit test has a dependency on `//chrome/install_static:install_static_util`, which has a public dependency on `//chrome/chrome_elf:nt_registry`

Reply:
we always want direct dependencies for headers. Deps of deps change and cause other problems so `//chrome/chrome_elf:nt_registry` should be added

Reply:
Added.